src/lib/config/defaultConfig.js

```diff
@@ -30,6 +31,7 @@ export const APP_CONFIG = {
   REST_ENDOINTS_BASE_URL: process.env.REACT_APP_INVENIO_REST_ENDPOINTS_BASE_URL,
   SUCCESS_AUTO_DISMISS_SECONDS: 10,
   DEFAULT_CURRENCY: DEFAULT_CURRENCY,
+  DEFAULT_LANGUAGES: [DEFAULT_LANGUAGE],
```

Comment:
I think it should be string here, not an array

Reply:
I thought maybe someone would like to have more than one default language, but I see your point, I will change it. Thanks!